actions/bulk_search_issues.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: bulk_search_issues
+runner_type: python-script
+description: "Receives list of Summary and a single jql query Substring. Joins each Summary with jql substring, and returns a list of  lists containing tickets found for each summary & substring pair, e.g in format: [ [ticket1,ticket2,ticket3], [], [ticket4] ]"
```

Comment:
I think this description is not quite clear on what this action is aiming to achieve that the existing `search_issues` action fails to do. Can you perhaps explain here or open an issue describing the use case?

Reply:
For example, if one wants to search existing issues based on following two queries:

1. {"type='Bug' AND project = "anyproj" AND summary ~ 'SummaryOne'"}
2.  {"type='Bug' AND project = "anyproj" AND summary ~ 'SummaryTwo'"}

So in above two jql queries, we want to search existing issues where we have different summaries for jql query string. I had a use case where I had multiple ( around 30+) summaries based on which I wanted to search existing issues so instead of executing workflow for each query separately(which was taking lot of time),
this workflow will take two inputs: 1. list of summary 2. subsequent jql substring
and then will search all issues based on those different jql queries(resulted due to different summaries) at one run and saved a good amount of time.

I would edit the description to make it more understandable.

Reply:
Sorry for being dense. It seems to me that this could be addressed with a more advanced jql query. Using your example:

```
{"type='Bug' AND project = "anyproj" AND (summary ~ 'SummaryOne') OR (summary ~ 'SummaryTwo')"}
```